### PR TITLE
Get docs working via Travis & github pages

### PR DESCRIPTION
**Pull request by rohanpm**
Basic skeleton of docs are in place.

Currently, the classes/methods appearing in "API reference" should
be considered as examples only.  I'm not sure exactly what should
go in there as it still seems a bit unclear what is and isn't
public API.  The main goal here is to get the doc building toolchain
up and running.

### CODE REVIEW

**Comment by rohanpm** [comment]
A GITHUB_TOKEN will need to be added to Travis CI env vars before the push from master can work.

**Comment by rohanpm** [comment]
Per discussion in https://github.com/release-engineering/ubi-population-tool/issues/9, it might make sense to drop all the references to API for now.

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'ubi-population-tool'
+copyright = '2018, Red Hat'
```
Comment (alexandrevicenzi) [inline docs/conf.py]: 2019 right?

Comment (rohanpm) [reply docs/conf.py]: Yep, fixed.

**Comment by rbikar** [comment]
Token added to travis, merging.

**Pull request merged**